src/macaron/slsa_analyzer/registry.py

```diff
@@ -579,5 +775,66 @@ def _should_skip_check(check: BaseCheck, results: dict[str, CheckResult]) -> Ski
 
         return None
 
+    def _get_check_tree_as_dict(self) -> CheckTree:
+        """Return a dictionary representation of the check relationships.
+
+        Returns
+        -------
+        CheckTree
+            A nested dictionary that represent the relationship between
+            checks. Each mapping (K, V) in the returned dictionary has K is the check id and
+            V is a dictionary contains the children of that check.
+
+        Examples
+        --------
+        Given the following checks and its relationships
+
+        .. code-block::
+
+            mcn_provenance_available_1
+            |-- mcn_provenance_level_three_1
+                |-- mcn_provenance_expectation_1
+            mcn_version_control_system_1
+            |-- mcn_trusted_builder_level_three_1
+
+        The result dictionary will be
```

Comment:
```suggestion
        The resulting dictionary will be:
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/254/commits/63b2883b703ee2b96584f00cb34f560f49a87db9